docs/rules.md

```diff
@@ -387,3 +387,4 @@ TODO: A lot of these are generated so this will need expanded with each unique c
 | :white_check_mark: | warning | Web extension | Temporary IDs can cause issues with identity.getRedirectURL | | | null | IDENTITY_GETREDIRECTURL |
 | :white_check_mark: | warning | Web extension | When default_locale is specified a matching messages.json must exist | | | null | NO_MESSAGES_FILE |
 | :white_check_mark: | warning | Web extension | When _locales directory exists, default_locale must exist | | | null | NO_DEFAULT_LOCALE |
+| :white_check_mark: | warning | Web extension | desc | | | | UNSAFE_VAR_ASSIGNMENT |
```

Comment:
"desc" doesn't seem very descriptive. This code also isn't being set right now so maybe this isn't needed?